{"uuid": "a4788d85-bae2-4369-9084-f2efaa06aef7", "children": ["6151e8a6-5d0a-4bb1-b2eb-0ca487add5f8"], "befores": [{"name": "get_browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: window was already closed\n  (Session info: chrome=94.0.4606.71)\n\n", "trace": "  File \"/Users/admin/PycharmProjects/PageObjectModelFramework/venv/lib/python3.9/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/admin/PycharmProjects/PageObjectModelFramework/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/admin/PycharmProjects/PageObjectModelFramework/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/admin/PycharmProjects/PageObjectModelFramework/Testcases/conftest.py\", line 46, in get_browser\n    driver.maximize_window()\n  File \"/Users/admin/PycharmProjects/PageObjectModelFramework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 737, in maximize_window\n    self.execute(command, params)\n  File \"/Users/admin/PycharmProjects/PageObjectModelFramework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/admin/PycharmProjects/PageObjectModelFramework/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1641495225175, "stop": 1641495266259}], "start": 1641495225175, "stop": 1641495266321}